From 061730cf6a2d6302b205f323545c4483bf3a8e23 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 31 Jul 2015 10:30:58 +0100
Subject: [PATCH] Remove !! from "No coretemp sysfs found"

Users are getting alarmed by the double exclamation marks in this
message, remove the !! exclamation marks

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/thd_engine_default.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/thd_engine_default.cpp b/src/thd_engine_default.cpp
index 1680920..d10ae59 100644
--- a/src/thd_engine_default.cpp
+++ b/src/thd_engine_default.cpp
@@ -185,7 +185,7 @@ int cthd_engine_default::read_thermal_sensors() {
 	}
 	if (index == current_sensor_index) {
 		// No coretemp sysfs exist, try hwmon
-		thd_log_warn("Thermal DTS: No coretemp sysfs found!!\n");
+		thd_log_warn("Thermal DTS: No coretemp sysfs found\n");
 	}
 	current_sensor_index = index;
 	// Add from XML sensor config
-- 
2.7.0.rc3

